kibana_logger/__init__.py

```diff
@@ -18,6 +21,7 @@ class KibanaLogger(object):
     CEE = "@cee: "
 
     def __init__(self, preset=None):
+        self.log_level = LogLevel[os.getenv('KIBANA_LOGGER_LOG_LEVEL', default='INFO')]
```

Comment:
now that I think about it,  it would be more optimized (especially for code called a lot of time) 

if instead of having a if in each function,  that  depending of the level , in the `__init__` you set function to `_do_nothing`  

like this 

```
if log_level > debug
      self.debug =  self._do_nothing
if log_level > info 
     self.info =  self._do_nothing 
etc.
```